plugins/modules/ovirt_vm.py

```diff
@@ -1731,6 +1731,18 @@ def post_present(self, entity_id):
         self.changed = self.__attach_graphical_console(entity)
         self.changed = self.__attach_host_devices(entity)
 
+        # Wait for lease
+        if self.param('lease'):
+            jobs_service = self._connection.system_service().jobs_service()
+            jobs = sorted(jobs_service.list(), reverse=True, key=lambda job: job.start_time)
+            if jobs:
+                wait(
+                    service=jobs_service.job_service(jobs[0].id),
+                    condition=lambda job: job.status == otypes.JobStatus.FINISHED,
+                    wait=self.param('wait'),
+                    timeout=self.param('timeout'),
+                )
+
```

Comment:
How can we found out the correct job if there are 2 parallel request to create a VM with lease?